<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navconnect - Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxFetchAiAircraft">
       <property name="toolTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer aircraft</string>
       </property>
       <property name="statusTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer aircraft</string>
       </property>
       <property name="text">
        <string>Fetch AI or multiplayer &amp;aircraft</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxOptionsUpdateRate">
       <property name="toolTip">
        <string>Allowed range is 50 milliseconds up to 2 seconds.
Little Navconnect fetches data from Flight Simulator and sends it to all clients (i.e. Little Navmap) using this time interval.</string>
       </property>
       <property name="statusTip">
        <string>Allowed range is 50 milliseconds up to 2 seconds.  Little Navconnect fetches data from Flight Simulator and sends it to all clients (i.e. Little Navmap) using this time interval.</string>
       </property>
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxHideHostname">
       <property name="toolTip">
        <string>Select this if you do not want to reveal your hostname or IP address in the log window.
This can be useful for streaming sessions where others can see your desktop.</string>
       </property>
       <property name="statusTip">
        <string>Select this if you do not want to reveal your hostname or IP address in the log window.\nThis can be useful for streaming sessions where others can see your desktop.</string>
       </property>
       <property name="text">
        <string>&amp;Hide hostname and IP in log window</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelOptionsPort">
       <property name="text">
        <string>&amp;Network Port:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinBoxOptionsPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelOptionsUpdateRate">
       <property name="text">
        <string>&amp;Update Time Interval:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinBoxOptionsUpdateRate</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxOptionsPort">
       <property name="toolTip">
        <string>This can normally be left at default.
Change this value only if another application occupies this port and
you get an error message like &quot;The bound address is already in use&quot;.</string>
       </property>
       <property name="statusTip">
        <string>This can normally be left at default.\nChange this value only if another application occupies this port and \nyou get an error message like &quot;The bound address is already in use&quot;.</string>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>51968</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxFetchAiShip">
       <property name="toolTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer ships. This option does not apply to X-Plane.</string>
       </property>
       <property name="statusTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer ships. This option does not apply to X-Plane.</string>
       </property>
       <property name="text">
        <string>Fetch AI or multiplayer</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxOptions">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxOptionsUpdateRate</tabstop>
  <tabstop>spinBoxOptionsPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
